src/components/InfoTooltip/InfoTooltip.css

```diff
@@ -0,0 +1,95 @@
+.info-tooltip-content {
```

Comment:
There are already some general CSS tooltip styles in `themes/default/tooltip`. I.e. much of this might not be needed. Also to point out that the demo styles from the Radix docs are referencing color vars we don't actually have!

I believe it should actually be fine to simply remove this stylesheet altogether. As long `tooltip`, `tooltip-arrow` and `tooltip-content` classes to your tooltip, it should pick up the base styles we already have.

Reply:
I have tended to add the custom CSS.  I do believe I have changed the vars to match ours when we actually used the class, but I see I left some unused classes from the demo :-/